src/app/services/tests/ConfigRoomService.spec.ts

```diff
@@ -35,77 +35,69 @@ describe('ConfigRoomService', () => {
 
             ],
         }).compileComponents();
-        dao = TestBed.inject(ConfigRoomDAO);
+        configRoomDAO = TestBed.inject(ConfigRoomDAO);
         service = TestBed.inject(ConfigRoomService);
     }));
     it('should create', fakeAsync(() => {
         expect(service).toBeTruthy();
     }));
     it('read should be delegated to ConfigRoomDAO', fakeAsync(async() => {
         // Given a ConfigRoomService
-        spyOn(dao, 'read').and.resolveTo(MGPOptional.of(ConfigRoomMocks.INITIAL));
+        spyOn(configRoomDAO, 'read').and.resolveTo(MGPOptional.of(ConfigRoomMocks.INITIAL));
```

Comment:
Tiens, in fine, c'est quoi notre convention ici, on les nomme d'une façon différente en fonction de si ils sont SUT ou pas ?

Reply:
on partait sur xxxDAO non ? Mais on a pas arrêté la décision !